### DISCUSSION TITLE
Doris 1.2.3版本。从checkpoint启动出现 [ANALYSIS_ERROR]errCode = 2, detailMessage = current running txns on db 555463 is 100, larger than limit 100

### DISCUSSION THREAD

**Discussion by victoyzz** [apache/doris#19023] [A - General / Q&A]
执行命令，show proc "/transactions/555463/running"。看到很多
1、大量进程：wait for publishing partition
2、一两个进程为：publish on tablet failed. succeed replica num 0 less than quorum 2
3、还有很多进程卡在PRECOMMITTED

此类问题的的排查思路是？

**Comment by LemonLiTree** [comment]
Increase the parameter of be ： max_running_txn_num_per_db
